Code/GraphMol/Descriptors/AtomicEnvironmentVector.cpp

```diff
@@ -796,7 +787,7 @@ ArrayXXd AtomicEnvironmentVector(
     index.row(i) = triuIndices(species12_(0, i), species12_(1, i));
   }
   // The constant 10 comes from 10 pairs that can be formed
-  index = index + (centralAtomIndexArr.array() * 10).array();
+  index = index + (centralAtomIndexArr * 10);
```

Comment:
Can you use `+=` here to avoid another copy?
```suggestion
  index += centralAtomIndexArr * 10;
```